web/client/codechecker_client/cmd/store.py

```diff
@@ -234,6 +235,10 @@ def collect_file_hashes_from_plist(plist_file):
         try:
             files, reports = plist_parser.parse_plist_file(plist_file)
 
+            rep_hash = [rep.report_hash == '0' for rep in reports]
+            if all(rep_hash):
```

Comment:
Can not we just use `any` to decide that we should generate a hash for a report?
```suggestion
            if any(rep.report_hash == '0' for rep in reports):
```

Reply:
I wanted to stay on the safer side and only rewrite the hashes if all of them is '0' in the plist reports. If not all of them is '0' it will overwrite the already existing value in that case.